src/components/Glp/GlpSwap.js

```diff
@@ -432,7 +441,13 @@ export default function GlpSwap(props) {
 
     if (isBuying) {
       const swapTokenInfo = getTokenInfo(infoTokens, swapTokenAddress);
-      if (swapTokenInfo && swapTokenInfo.balance && swapAmount && swapAmount.gt(swapTokenInfo.balance)) {
+      if (
+        !savedShouldDisableOrderValidation &&
```

Comment:
order validation in glp swap?

Reply:
if i understand correctly, you're using this setting for testing.

i think it's better to create separate setting for this, otherwise it might confuse others who will work with that code

Reply:
We are using this setting for testing and removing the available balance check. It only works on the development. In production the option will not even be visible. It also helps xhiroz to test the app properly. Otherwise extreme conditions are hard to test.

We have used this at multiple place in SwapBox. Maybe we can improve the name to say something like test, dev etc. Please let me know your thoughts.

Reply:
yup, i understand how it works. i mean it's name may be confusing in this context. initially it was added to disable orders validation and in UI in settings it's "Disable order validations". and i guess in master it is used for order price validation only? 

i'm not sure if it makes sense to create new flags/checkboxes for each new case, so let's just rename it (and change text in UI). maybe something like "savedShouldDisableValidationForTesting" or something like this? do you have better name in mind?

Reply:
The name makes sense! I will update to `savedShouldDisableValidationForTesting`